formats/czi-extract-plugin/tests/fixture.py

```diff
@@ -0,0 +1,44 @@
+"""Test fixtures.
+
+Set up all data used in tests.
+"""
+import shutil
+import tempfile
+from pathlib import Path
+from typing import Union
+
+import pytest
+import requests
+
+
+def clean_directories() -> None:
+    """Remove all temporary directories."""
+    for d in Path(".").cwd().iterdir():
+        if d.is_dir() and d.name.startswith("tmp"):
+            shutil.rmtree(d)
+
+
+@pytest.fixture()
+def output_directory() -> Union[str, Path]:
+    """Create output directory."""
+    return Path(tempfile.mkdtemp(dir=Path.cwd()))
```

Comment:
Do not pass the `dir` parameter here. `tempfile` will automatically determine the correct place for the directory. Further, when the tests end (regardless of whether they succeeded or failed), the temp directories will automatically be cleaned up.